### ISSUE TITLE
Complex pages

### ISSUE THREAD

**Issue by ValWood** [pombase/website#294]
we want these eventually. We really need complex IDs though.

Maybe we can do a "complex" display were we just display all of the annotation  with a specific GO complex ID as an interim if its quick.

So, it would look  a little like a  publication page

http://pombase2.aska.gen.nz/reference/PMID%3A2569363

but instead of
"Annotations from this publication"
it would be
"annotations to this complex"

This would be useful/interesting  to see phenotypes like this because  complex members usually pheno-copy each other. ....and obviously we would expect similar processes to.

It would be handy to also be able to see all of the extensions for a complex together because 
sometimes the extensions are attached to different subunits....

**Comment by mah11** [comment]
How will this differ from an ontology term page (whether complexes continue to live in GO CC or elsewhere)?

**Comment by ValWood** [comment]
Because an ontology term page only shows annotations to the complex term?

We want  a page which shows "all annotation to the complex" 
functions, processes, phenotypes etc.
They are very different ...

**Comment by kimrutherford** [comment]
> we want these eventually. We really need complex IDs though.

We have the complex IDs and names in Chado now.

> "annotations to this complex"

Would that be all the annotations for all genes in the complex?

**Comment by ValWood** [comment]
yes, but we might need to think of a way to present this so it isn't a completely swamping list...

It will be huge for some complexes...

**Comment by kimrutherford** [comment]
Does every complex have a GO term like this?: https://www.pombase.org/term/GO:1990130

**Comment by ValWood** [comment]
Not all becasue GO don't create terms for every dimer etc (a lot of transcription factor dimers)

But  I think we can use the GO complex file as a proxy. If we needed a dimer for modelling it would need to go into GO.

this list removes unnecessary ancestors and grouping terms
https://www.pombase.org/data/annotations/Gene_ontology/GO_complexes/
